apps/frontend/src/libs/components/sidebar/styles.module.css

```
@@ -270,11 +267,26 @@
 		flex-grow: 3;
 	}
 
-	.log-out-btn :last-child {
-		display: flex;
-		flex-direction: column;
-		gap: 0;
-		line-height: 24px;
+	.link-title,
+	.log-out-icon ~ :last-child {
```

```suggestion
	.log-out-icon {
```
do not use this hard selector. add an additional classes or something